cmake/scripts/ios/ArchSetup.cmake

```diff
@@ -34,7 +34,7 @@ list(APPEND DEPLIBS "-framework CoreFoundation" "-framework CoreVideo"
                     "-framework CFNetwork" "-framework CoreGraphics"
                     "-framework Foundation" "-framework UIKit"
                     "-framework CoreMedia" "-framework AVFoundation"
-                    "-framework VideoToolbox")
+                    "-framework VideoToolbox" "-lresolv")
```

Comment:
who provides this library, no probing required?

Reply:
ios sdk